src/emulator/auth/operations.ts

```diff
@@ -206,6 +208,7 @@ function lookup(
 
   if (ctx.security?.Oauth2) {
     if (reqBody.initialEmail) {
+      // TODO: This is now possible. See ProjectState.getUserByInitialEmail.
```

Comment:
This can be done with a follow up PR with test cases.

Reply:
Yup. That's why I left it as it is. Open for external contributions too.